tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderingPartySteps.cs

```diff
@@ -75,46 +75,36 @@ public void GivenAnOrderPartyUpdateRequestExistForOrderID(string orderId)
             SetOrganisationPartyPayloadByOrderId(_context, orderId, new OrganisationPartyPayload());
         }
 
-        [Given(@"the update request for order ID (.*) has a organisation contact")]
-        public void GivenTheUpdateRequestForOrderIdcHasAOrganisationContact(string orderId, Table table)
+        [Given(@"the update request for order ID (.*) has a contact")]
+        public void GivenTheUpdateRequestForOrderIdcHasAContact(string orderId, Table table)
         {
             var payload = GetOrganisationPartyPayloadByOrderId(_context, orderId);
             payload.PrimaryContact = table.CreateInstance<ContactPayload>();
         }
 
-        [Given(@"the order party update request for order ID (.*) has a organisation address")]
-        public void GivenTheOrderPartyUpdateRequestForOrderIdcHasAOrganisationAddress(string orderId, Table table)
+        [Given(@"the order party update request for order ID (.*) has a address")]
+        public void GivenTheOrderPartyUpdateRequestForOrderIdcHasAAddress(string orderId, Table table)
         {
             var payload = GetOrganisationPartyPayloadByOrderId(_context,orderId);
-            if (payload.Organisation == null)
-            {
-                payload.Organisation = new OrganisationPayload();
-            }
-            payload.Organisation.address = table.CreateInstance<AddressPayload>();
+
+            payload.Address = table.CreateInstance<AddressPayload>();
         }
 
-        [Given(@"the order party update request for order ID (.*) has a organisation Name of (.*)")]
-        public void GivenTheOrderPartyUpdateRequestForOrderIdcHasAOrganisationNameOfTestCareCenter(string orderId, string name)
+        [Given(@"the order party update request for order ID (.*) has a Name of (.*)")]
+        public void GivenTheOrderPartyUpdateRequestForOrderIdcHasANameOfTestCareCenter(string orderId, string name)

```

Comment:
The method name contains Idc, is that right? Actually noticed this in a few more places, sorry!

Reply:
👍